modules/ROOT/pages/changelog.adoc

```diff
@@ -4,6 +4,86 @@
 
 NOTE: This is the {productname} Community version changelog. For information about the latest {cloudname} or {enterpriseversion} Release, see: xref:release-notes.adoc[{productname} Release Notes].
 
+== 6.5 - 2023-06-05
```

Comment:
Reminder to update this to match the release-date.

Reply:
Done.